numpy/core/src/multiarray/methods.c

```diff
@@ -2777,6 +2784,9 @@ NPY_NO_EXPORT PyMethodDef array_methods[] = {
     {"__array_prepare__",
         (PyCFunction)array_preparearray,
         METH_VARARGS, NULL},
+    {"__array_finalize__",
+        (PyCFunction)array_finalizearray,
+        METH_VARARGS, NULL},
```

Comment:
```suggestion
        METH_O, NULL},
```
[`METH_O`](https://docs.python.org/3/c-api/structures.html#METH_O) should be useable here, right? Considering it only has a single positional-only argument.

Reply:
Yes, that makes sense. adopted.